adapters/src/main/kotlin/com/krzykrucz/elesson/currentlesson/topic/adapters/rest/topicDto.kt

```diff
@@ -0,0 +1,6 @@
+package com.krzykrucz.elesson.currentlesson.topic.adapters.rest
+
+import com.krzykrucz.elesson.currentlesson.domain.startlesson.LessonIdentifier
+import com.krzykrucz.elesson.currentlesson.domain.topic.domain.TopicTitle
+
+data class ChooseTopicDto(val lessonIdentifier: LessonIdentifier, val topicTitle: TopicTitle)
```

Comment:
I'm not Kotlin expert but shouldn't this file name use CamelCase if it contains only one class? btw. could you please provide some references to the used naming convention when it comes to files containing multiple units (data classes, functions etc.) of the same purpose (dtos, helpers etc.) ? Just would like to learn sth :wink:

Reply:
We use convention that all files should be upper case